### ISSUE TITLE
Investigate removing "None" nodes.

### ISSUE THREAD

**Issue by kgroble** [RHIT-XPrize/rhit-xprize-neural-network#22]
We currently have "None" outputs for the neural networks which determine color and letter. Investigate removing these and interpreting low probabilities across the board of concrete results as "none".

**Comment by rbubulka** [comment]
while it would be pretty simple to just take out the None node and the softmax to adapt to result in low probabilities, the real gain in doing so would be from implementing outlier detection and novelty learning which I do not believe we have time to properly implement with an understanding of what we are doing. Here is two links to the scikit-learn documentation for outlier detection and the commit where they first started implementing it, scikit learn is a tensorflow intermediary similar to keras. 

http://scikit-learn.org/stable/modules/outlier_detection.html
https://github.com/scikit-learn/scikit-learn/pull/10700/files

**Comment by rbubulka** [comment]
I believe the best course of action is to ignore this route for now and hand it off to the next team
